src/components/ConversationsList/ConversationTile.tsx

```diff
@@ -1,36 +1,36 @@
 import { Divider, Flex, LinkBox, LinkOverlay, Text } from '@chakra-ui/react'
-import type { Conversation } from '@xmtp/xmtp-js'
-import React from 'react'
+import React, { useMemo } from 'react'
 import Emoji from 'react-emoji-render'
 import { formatDate, truncate } from '../../helpers'
-import useConversation from '../../hooks/useConversation'
+import useChat from '../../hooks/useChat'
 import Address from '../Address'
 import Avatar from '../Avatar'
 
 type ConversationTileProps = {
-  conversation: Conversation
+  peerAddress: string
   isSelected: boolean
   onClick?: () => void
 }
 
 // TODO: add design for `isSelected`
 const ConversationTile = ({
-  conversation,
+  peerAddress,
   isSelected,
```

Comment:
This `isSelected` prop is unused. Possibly intentional to go with the `TODO` comment above though?

Reply:
yes this is related to the TODO, we should design this part
Added an issue for that https://github.com/liteflow-labs/chat/issues/17